docroot/themes/humsci/humsci_basic/humsci_basic.theme

```diff
@@ -37,3 +37,14 @@ function humsci_basic_preprocess_field(&$variables, $hook) {
     }
   }
 }
+
+/**
+ * Implements hook_preprocess_page().
+ */
+function humsci_basic_preprocess_page(&$vars) {
+  // Variant setting for the brand bar.
+  $bbv = theme_get_setting('brand_bar_variant');
+  if ($bbv !== "none") {
+    $vars['brand_bar_variant'] = 'su-brand-bar--' . $bbv;
```

Comment:
can we switch this to `$vars['brand_bar_variant_classname']` I think it's clearer (and I just suggested that to heather too). You will also need to make sure to pass this new classname to the block in page.html.twig :)